app/assets/javascripts/discourse/app/routes/build-category-route.js

```diff
@@ -137,11 +101,14 @@ export default (filterArg, params) => {
     },
 
     _retrieveTopicList(category, transition, modelParams) {
-      const listFilter = `c/${Category.slugFor(category)}/${
-          category.id
-        }/l/${this.filter(category)}`,
-        findOpts = filterQueryParams(modelParams, params),
-        extras = { cached: this.isPoppedState(transition) };
+      const findOpts = filterQueryParams(modelParams, params);
+      const extras = { cached: this.isPoppedState(transition) };
+
+      let listFilter = `c/${Category.slugFor(category)}/${category.id}`;
```

Comment:
Is this used for the preload store key? If yes, does this match the server method `TopicList#preload_key`?

Reply:
Nice catch, I pushed a fix to make one less request for /none pages.